java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -672,4 +672,124 @@ CompletableFuture<Object[][]> xpending(
             StreamRange end,
             long count,
             StreamPendingOptions options);
+
+    /**
+     * Returns information about the stream stored at key <code>key</code>.<br>
+     * To get more detailed information use {@link #xinfoStreamFull(String)} or {@link #xinfoStreamFull(String, int)}.
+     *
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @return Stream information.
+     * @example <pre>{@code
+     * // example of using the API:
+     * Map<String, Object> response = client.xinfoStream("myStream").get();
+     * // the response contains data in the following format:
+     * Map<String, Object> data = Map.of(
+     *     "length", 4L,
+     *     "radix-tree-keys", 1L,
+     *     "radix-tree-nodes", 2L,
+     *     "last-generated-id", "1719877599564-0",
+     *     "max-deleted-entry-id", "0-0",
+     *     "entries-added", 4L,
+     *     "recorded-first-entry-id", "1719710679916-0",
+     *     "groups", 1L,
+     *     "first-entry", Map.of(
+     *         "1719710679916-0",
+     *         new String[][] {
+     *             { "foo", "bar" },
+     *             { "foo", "bar2" },
+     *             { "some_field", "some_value" }
+     *         }),
+     *     "last-entry", Map.of(
+     *         "1719877599564-0",
+     *         new String[][] {
+     *             { "e4_f", "e4_v" }
+     *         })
+     * );
+     * // "first-entry" and "last-entry" could be both `null` if stream is empty
+     * }</pre>
+     */
+    CompletableFuture<Map<String, Object>> xinfoStream(String key);
+
+    /**
+     * Returns verbose information about the stream stored at key <code>key</code>.<br>
+     * The output is limited by first <code>10</code> PEL entries.
```

Comment:
Just double checking, is this only applicable when the FULL arg is passed? If its applicable above as well we should add this above too.

Reply:
As I understand docs, yes:
> The COUNT option can be used to limit the number of stream and PEL entries that are returned (The first <count> entries are returned). The default COUNT is 10 and a COUNT of 0 means that all entries will be returned (execution time may be long if the stream has a lot of entries).

I'll test it with 11 entries if I won't forget (UPD already forgot).